site/docs/guides/admin/servers/configuring-an-open-lineage-server.md

```diff
@@ -82,13 +98,14 @@ For open lineage server following can be configured:
 | Property | Description | Is mandatory |
 |---|---|---|
 `lineageGraphConnection` | OCF configuration object that defines the Graph store connector type used. See [open-lineage-janus-connector](/connectors/governance-daemon/open-lineage-janus-connector) for more details. | Yes |
-`accessServiceConfig.serverName` | the name of the metadata server where paired Asset Lineage OMAS is running. | Yes
+`accessServiceConfig.serverName` | the name of the metadata server where paired Asset Lineage OMAS is running. | Yes |
 `accessServiceConfig.serverPlatformUrlRoot` | The URL of the OMAG server platform running the metadata server where paired Asset Lineage OMAS is running. Also see [start-up information](#start-up-information) section. | Yes |
 `accessServiceConfig.user` | The username to access the server running Asset Lineage OMAS. | Yes |
 `accessServiceConfig.password` | The user password to access the server running Asset Lineage OMAS. Can be left out for non-secured access. | No |
+`assetLineageTopicConnectionOverride` | Configuraton oject that provides the Asset Lineage OMAS kafka topic properties. If provided, it will override the default configuraiton  | No |
```

Comment:
Spelling mistakes: `Configuration` (x2) and `object`

`Configuration object` should probably be `[Connection object](/concepts/connection)`